proposals/readonly-ref.md

```diff
@@ -460,6 +460,7 @@ The main competing design is really "do nothing".
 * Conversions at 'in' call sites.
 * Need to agree on metadata representation.
 * Changes to the Verification rules
+* Spec 7.5.3.1 would have to change to reflect that ref extension methods arguments are now passed without using a 'ref' keyword.
```

Comment:
Considering the confusing situation with C# spec versions, wouldn't a better option be to link to the relevant section of the spec in this repo, instead of using section number in an unspecified version of the spec?

I.e.:

> Spec section [Applicable function member](https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#applicable-function-member) would have to change to reflect that ref extension methods arguments are now passed without using a 'ref' keyword.